app/models/place.rb

```diff
@@ -29,4 +33,52 @@ def self.prefectures_with_key
       }
     end
   end
+
+  def self.create_index!(options={})
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for create_index! is too high. [16.16/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Method has too many lines. [35/10] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)
* Surrounding space missing in default value assignment. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-around-equals)